src/components/InputSection.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
```

Comment:
We should not have to import React with a Functional Component; only the hooks if we need them

Reply:
Yes. Good call. We can remove that too.